src/resources/dicts/zh-cn.edn

```diff
@@ -49,11 +81,22 @@
  :discourse-title "我们的论坛"
  :export-edn "导出为 EDN"
  :export-public-pages "导出公开页"
+ :export-transparent-background "透明背景"
+ :export-copy-to-clipboard "复制到剪贴板"
+ :export-copied-to-clipboard "已复制到剪贴板"
+ :export-save-to-file "保存到文件"
  :content/copy-block-emebed "复制内嵌块"
  :help/awesome-logseq "绝妙的 Logseq"
  :help/forum-community "论坛讨论"
+ :help/title-usage "使用说明"
+ :help/title-community "社区"
+ :help/title-development "开发"
+ :help/title-about "关于"
+ :help/title-terms "条款"
  :linked-references/filter-search "在链接的页面内搜索"
  :page/copy-page-url "复制页面 URL"
+ :page/slide-view "幻灯片视图"
+ :page/slide-view-tip-go-fullscreen (fn [] [[:span.opacity-70 "小技巧: 按 "] [:code "f"] [:span.opacity-70 " 进入全屏"]])
```

Comment:
No idea of if it suitable to mix style, markup and translation

Reply:
I used to have it separated, but someone suggest me to move it together. So do you guys have a same standard?

Reply:
@logseq-cldwalker, This is suitable right?

Reply:
@andelf It should generally be avoided, but there is no other way to properly handle those cases.
See the related tongue issue here https://github.com/tonsky/tongue/issues/11
It's the equivalent of https://react.i18next.com/latest/trans-component

Reply:
@andelf Unfortunately this is needed for complete sentence translations as languages have different ordering of words. It's mentioned for contributors in https://github.com/logseq/logseq/blob/master/docs/contributing-to-translations.md#editing-tips. I can mention it in https://github.com/logseq/logseq/blob/master/docs/dev-practices.md#translations for reviewers (core team) after the PR is merged